src/game/logic/object/object.cpp

```diff
@@ -367,3 +369,34 @@ void Object::Set_Team(Team *team)
     Call_Method<void, Object, Team *>(PICK_ADDRESS(0x00546800, 0x007CFB7A), this, team);
 #endif
 }
+
+void Object::Heal_Completely()
+{
+#ifdef GAME_DLL
+    Call_Method<void, Object>(PICK_ADDRESS(0x00548430, 0x007D1800), this);
+#endif
+}
+
+float Object::Get_Vision_Range() const
+{
+#ifdef GAME_DEBUG_STRUCTS
+    if (g_theWriteableGlobalData->m_debugVisibility) {
+        Vector3 v(m_visionRange, 0.0f, 0.0f);
+
+        for (int i = 0; i < g_theWriteableGlobalData->m_debugVisibilityTileCount; i++) {
+            float f1 = i * 1.0f / g_theWriteableGlobalData->m_debugVisibilityTileCount;
+            float angle = (f1 + f1) * 3.1415927;
```

Comment:
Do we have a matching PI constant for this?

Reply:
We do. Fixed.